kubeportal/models.py

```diff
@@ -176,12 +176,15 @@ def has_access_requested(self):
         return result
 
     @transition(field=state, source=[UserState.NEW, UserState.ACCESS_REQUESTED, UserState.ACCESS_APPROVED, UserState.ACCESS_REJECTED], target=UserState.ACCESS_REQUESTED)
-    def send_access_request(self, request):
+    def send_access_request(self, request, *args, **kwargs):
```

Comment:
This looks like overkill. Wouldn't it be good enough to have a new parameter "administrator=None" for the method?. The default ("None") stands for sending it to all, setting it chooses one.